src/view_curses.cc

```diff
@@ -322,8 +324,22 @@ view_curses::mvwattrline(WINDOW* window,
             int awidth = attr_range.length();
             nonstd::optional<char> graphic;
 
+            short graphic_inside_style = 0;
+
             if (iter->sa_type == &VC_GRAPHIC) {
                 graphic = iter->sa_value.get<int64_t>();
```

Comment:
I think the problem is only that `attrs` is set on the other branches, but not this one.  Adding `attrs = text_attrs{}` might be enough to fix it.  That will set it to an empty value.

Reply:
Not sure about this, especially considering the test on the REVERSE status below. I'll do a few tests.

Reply:
Ok, I've tried your suggestion and avoided performing the "secondary" iteration. The results are similar, although this time the VC_GRAPHIC "bars" are not REVERSE-d (only) when overlapping a colored bar (since there's no way to know whether there's an overlap or not), hence they are less eye-catching compared to the screenshots above; I personally prefer the column-dividing bars to be highlited by the REVERSE-al, even at a (slight?) computational cost. What do you think?

Reply:
With the only change being this:

```diff
diff --git a/src/view_curses.cc b/src/view_curses.cc
index 78a61bc4..2e4664c9 100644
--- a/src/view_curses.cc
+++ b/src/view_curses.cc
@@ -324,6 +324,7 @@ view_curses::mvwattrline(WINDOW* window,
 
             if (iter->sa_type == &VC_GRAPHIC) {
                 graphic = iter->sa_value.get<int64_t>();
+                attrs = text_attrs{};
             } else if (iter->sa_type == &VC_STYLE) {
                 attrs = iter->sa_value.get<text_attrs>();
             } else if (iter->sa_type == &SA_LEVEL) {
```

The result of a `SELECT * FROM syslog_log` looks like the following, which seems right...

<img width="780" alt="Window 2022-09-04 at 13 59 02" src="https://user-images.githubusercontent.com/107532/188333302-ef6c2b57-5c84-428d-8dd6-a5cf85924709.png">

Reply:
@tstack Well, it is, as I mentioned. The only difference is the appearance of the bar characters, since they are similar to the surrounding text in your screenshot. See the screenshot below for a comparison to yours.
![Screenshot from 2022-09-04 18-48-25](https://user-images.githubusercontent.com/69568/188333628-57f2aeda-f98f-493c-9d9b-c77185435f39.png)
As I mentioned, it's mostly a "visual" choice, the one I proposed coming at a somewhat higher computational cost, although I don't know how higher. I'll leave to you the choice. :)

Reply:
Unless I force A_REVERSE to __every__ VC_GRAPHIC element, I guess it could work...

Reply:
Nope, after a few tries I came to agree with your proposal @tstack . Rewriting the commit in a few seconds...

Reply:
Oh, I'm sorry, you wanted the bars to not be reverse-video...  I missed that.

Reply:
@tstack no problem. I am having problems with the correct terminology due to my shallow grasp of the concepts, annoyingly.

Reply:
Anyway I think it works well enough like this. If I find the bars not to be "flashy" enough, I'll propose a change for that later on.

Reply:
> If I find the bars not to be "flashy" enough, I'll propose a change for that later on.

I don't use the charting features very much, so I'm open to ideas.